src/graphql/types/object/wallet-contact.ts

```diff
@@ -10,6 +15,33 @@ const WalletContact = new GT.Object({
     transactionsCount: {
       type: GT.NonNull(GT.Int),
     },
+
+    transactions: {
+      type: TransactionConnection,
+      args: connectionArgs,
+      resolve: async (source, args, { domainUser }) => {
+        const contactWalletName = source.walletName as WalletName
+
+        // TODO: figure out what to do here when we have multiple accounts
+        const account = await Accounts.getAccount(domainUser.id)
+
+        if (account instanceof Error) {
+          throw account
+        }
+
+        const { result: transactions, error } =
+          await Wallets.getAccountTransactionsForContact({
+            account,
+            contactWalletName,
+          })
```

Comment:
now we have two return patterns :( or what is the purpose of PartialResult?

Reply:
It's not needed for this PR but I kept it similar to `getTransactionsForWallet` that's used for the main `transactions` field. 

@bodymindarts should we keep it here or remove it?